modules/installation-machines-bare-metal.adoc

```diff
@@ -0,0 +1,103 @@
+= Creating the cluster
+
+To create the {product-title} cluster, you provision machines by using the Ingition config
+files that you generated with the installation program.
+
+.Prerequisites
+
+* Your machines must have internet access.
+* Create the required infrastructure for the cluster.
+* You obtained the installation program and generated the Ignition config files
+for your cluster.
+
+.Procedure
+
+. Obtain the {op-system-first} images.
+
+. Use the the ignition config files that you generated by using the installation
+program to create the required bootstrap, worker, and master nodes.
+
+. Monitor for `bootstrap-complete`:
++
+----
+$ openshift-install --dir test-bare-metal wait-for bootstrap-complete
+INFO Waiting up to 30m0s for the Kubernetes API at https://api.test.example.com:6443...
+INFO API v1.12.4+c53f462 up
+INFO Waiting up to 30m0s for the bootstrap-complete event...
+----
++
+The command succeeds when the Kubernetes APIServer signals that it has been
+bootstrapped on the control plane machines.
+
+. Remove the bootstrap machine from the load balancer.
++
+[IMPORTANT]
+====
+You must remove the bootstrap machine from the load balancer at this point.
+====
+
+. Confirm that the cluster recognizes the machines:
++
+----
+$ oc get nodes
+
+NAME      STATUS    ROLES   AGE  VERSION
+master-0  NotReady  master  50s  v1.12.4+c53f462
+master-1  NotReady  master  63s  v1.12.4+c53f462
+master-2  NotReady  master  64s  v1.12.4+c53f462
+worker-0  NotReady  worker  76s  v1.12.4+c53f462
+worker-1  NotReady  worker  70s  v1.12.4+c53f462
+worker-2  NotReady  worker  69s   v1.12.4+c53f462
+----
++
+The output lists all of the machines that you created.
+
+. Review the pending certificate signing requests (CSRs) and ensure that the
+requests are for the machines that you added to the cluster:
++
+----
+$ oc get csr
+----
+
+. Approve the CSRs for your cluster machines. You can approve them individually
+or, if all the CSRs are valid, approve them all by running the following
+command:
++
+----
+$ oc get csr -ojson | jq -r '.items[] | select(.status == {} ) | .metadata.name' | xargs oc adm certificate approve
+----
+
+. Watch the cluster components come online:
++
+----
+$ watch -n5 oc get clusteroperators
+----
+
+. Provide storage to the image registry.
+
+** For non-production clusters, you
+can set the image registry to an empty directory:
++
+----
+$ oc patch configs.imageregistry.operator.openshift.io cluster --type merge --patch '{"spec":{"storage":{"filesystem":{"volumeSource": {"emptyDir":{}}}}}}'
+----
+
+** For production clusters, you must configure either an Amazon Web Services
+S3 bucket or a Persistent Volume Claim (PVC) to provide registry storage. The
+PVC must be backed a RWX Persistent Volume on a production-ready file server or
+service that is sufficiently performant, redundant, and backed up.
+
+. Monitor for cluster completion:
++
+----
+$ ./openshift-install wait-for cluster-ready
```

Comment:
renamed to `wait-for install-complete`
https://github.com/openshift/installer/pull/1585

Reply:
I'm changing this here and on the AWS UPI PR. Thank you!